### PR TITLE
[zh] Translate blog/storage-in-tree-to-csi-migration-status-update

### LINKED ISSUE #31046: [zh] Translate blog/_posts/2021-12-10-csi-migration-status.md into Chinese
Hi, I'm interested to translate the "2021-12-10-csi-migration-status.md" page.

This is a Feature Request:

translate `en/blog/_posts/2021-12-10-csi-migration-status.md` into Chinese.

What would you like to be added:

zh/blog/_posts/2021-12-10-csi-migration-status.md

/language zh
/kind feature

### PR DESCRIPTION
**Pull request by IceCodeNew**
/language zh
fix #31046

### CODE REVIEW

File: content/zh/blog/_posts/2021-12-10-csi-migration-status.md [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+
+The Kubernetes v1.21 release deprecated the `CSIMigration{provider}Complete` feature flags, and stopped honoring them. In their place came new feature flags named `InTreePlugin{vendor}Unregister`, that replace the old feature flag and retain all the functionality that `CSIMigration{provider}Complete` provided.
+-->
+### 新的特性门控（feature gate）  {#new-feature-gates}
+Kubernetes v1.21 弃用了 `CSIMigration{provider}Complete` 特性参数（feature flag），它们不再生效。取而代之的是名为 `InTreePlugin{vendor}Unregister` 的新特性参数，它们保留了 `CSIMigration{provider}Complete` 提供的所有功能。
+
+<!---
+`CSIMigration{provider}Complete` was introduced before as a supplementary feature gate once CSI migration is enabled on all of the nodes. This flag unregisters the in-tree storage plugin you specify with the `{provider}` part of the flag name.
+-->
+`CSIMigration{provider}Complete` 是作为 CSI 迁移功能在所有节点上启用后的补充特性门控于之前引入的。这个参数可反注册参数名称中 `{provider}` 部分所指定的树内存储插件。
```
Comment (tengqm) [inline content/zh/blog/_posts/2021-12-10-csi-migration-status.md:86]: unregister --> 注销

File: content/zh/blog/_posts/2021-12-10-csi-migration-status.md [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+
+<!---
+When you enable that feature gate, then instead of using the in-tree driver code, your cluster directly selects and uses the relevant CSI driver. This happens without any check for whether CSI migration is enabled on the node, or whether you have in fact deployed that CSI driver.
+-->
+当你启用该特性门控时，你的集群不再使用树内驱动代码，而是直接选择并使用相应的 CSI 驱动。同时，这也将不再检查节点上 CSI 迁移功能是否启用，以及 CSI 驱动是否实际部署。
```
Comment (tengqm) [inline content/zh/blog/_posts/2021-12-10-csi-migration-status.md:91]:
```suggestion
当你启用该特性门控时，你的集群不再使用树内驱动代码，而是直接选择并使用相应的 CSI 驱动。同时，集群并不检查节点上 CSI 迁移功能是否启用，以及 CSI 驱动是否实际部署。
```

File: content/zh/blog/_posts/2021-12-10-csi-migration-status.md [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+| AWS EBS          | 1.14  | 1.17                      | 1.23                 | 1.24 (Target) | 1.26 (Target)                   |
+| GCE PD           | 1.14  | 1.17                      | 1.23                 | 1.24 (Target) | 1.26 (Target)                   |
+| OpenStack Cinder | 1.14  | 1.18                      | 1.21                 | 1.24 (Target) | 1.26 (Target)                   |
+| Azure Disk       | 1.15  | 1.19                      | 1.23                 | 1.24 (Target) | 1.26 (Target)                   |
+| Azure File       | 1.15  | 1.21                      | 1.24 (Target)        | 1.25 (Target) | 1.27 (Target)                   |
+| vSphere          | 1.18  | 1.19                      | 1.24 (Target)        | 1.25 (Target) | 1.27 (Target)                   |
+| Ceph RBD         | 1.23  |
+| Portworx         | 1.23  |
+-->
+| 驱动 | Alpha | Beta (in-tree deprecated) | Beta (on-by-default) | 正式发布 | 目标：移除“树内存储插件” |
```
Comment (tengqm) [inline content/zh/blog/_posts/2021-12-10-csi-migration-status.md:180]:
```suggestion
| 驱动 | Alpha | Beta (启用树内插件) | Beta (默认启用) | 正式发布 | 目标：移除“树内存储插件” |
```

**Review by tengqm — COMMENTED** [review]
谢谢，译文质量很高！！

**Comment by mayocream** [comment]
Great first contribution! 🎉

You can follow [this guide](https://github.com/kubernetes/community/blob/master/CLA.md) to sign CLA.

**Review by mayocream — COMMENTED** [review]
/lgtm

**Pull request merged**